tests/foreman/ui/test_hostgroup.py

```diff
@@ -141,3 +153,73 @@ def test_positive_create_with_oscap_capsule(self):
                     oscap_capsule=self.sat6_hostname,
                 )
             self.assertIsNotNone(self.hostgroup.search(name))
+
+    @run_only_on('sat')
+    @tier1
+    def test_positive_create_with_activation_keys(self):
+        """Create new hostgroup with activation keys
+
+        @id: cfda3c1b-37fd-42c1-a74c-841efb83b2f5
+
+        @Assert: Hostgroup is created with activation keys
+        """
+        name = gen_string('alpha')
+        with Session(self.browser) as session:
+            make_hostgroup(
+                session,
+                name=name,
+                environment=self.env.name,
+                content_view=self.cv.name,
+                activation_key=self.ak.name,
+            )
+            self.assertIsNotNone(self.hostgroup.search(name))
+
+    @run_only_on('sat')
+    @tier1
+    def test_positive_check_activation_keys_autocomplete(self):
+        """Open Hostgroup New/Edit modal and verify that Activation Keys
+        autocomplete respects selected Content View
+
+        @id: 07906caf-871d-4d1f-8914-38027e71c16b
+
+        @Steps:
+
+        1. Create two content views A & B
+        2. Publish both content views
+        3. Create an activation key pointed to view A in Library
+        4. Create an activation key pointed to view B in Library
+        5. Go to the new Hostgroup page, select content view B & Library,
+            click on the activation key tab and click in the input box
```

Comment:
`self.hostgroup.click(locators['hostgroups.activation_keys'])`
`locators['hostgroups.activation_keys']` points to input field.